### PR TITLE
test:  text correction and additional test cases

### PR DESCRIPTION
**Pull request by malviyagaurav**

### CODE REVIEW

File: apps/desktop/e2e/automated/base.spec.ts [hunk 1]
```diff
@@ -19,110 +19,94 @@ test.afterEach(async () => {
...
   await screen.screenshot({ path: './screenshots/title.png' });
-  expect(title).toBe('Cypherock CySync');
+  expect(title).toBe('Cypherock cySync');
 });
 
 test('check first screen', async () => {
   const header = screen.getByRole('heading', {
     name: 'Ensure the following before you continue',
   });
   await expect(header).toHaveText(['Ensure the following before you continue']);
```
Comment (ujjwal-cyph) [inline apps/desktop/e2e/automated/base.spec.ts:33]: replace `toHaveText` with `toBeVisible`

Comment (ujjwal-cyph) [reply apps/desktop/e2e/automated/base.spec.ts:33]: at line #65 as well.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "cysync",
   "version": "1.0.0",
-  "description": "Cypherock CySync application",
```
Comment (ghost) [inline package.json:4]: No need to change the description of the package

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (malviyagaurav) [reply package.json:4]: resolved

**Pull request merged**